### DISCUSSION TITLE
How to enable importing WM8960 library

### DISCUSSION THREAD

**Discussion by teddokano** [micropython/micropython#12691] [m.iMX RT / Teensy 4.x] [answered]
How the `import wm8960` can be done? Is there any tutorial for trying I2S bus sample code on [this page](https://docs.micropython.org/en/latest/mimxrt/quickref.html#i2s-bus)?
I tried "import wm8960" in REPL and got following error. 
Is the `wm8960` needed to be installed?

```python
MicroPython v1.21.0 on 2023-10-06; i.MX RT1010 EVK with MIMXRT1011DAE5A
Type "help()" for more information.
>>> import wm8960
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: no module named 'wm8960'
>>> 
```

**Comment by robert-hh** [comment, marked as answer]
yes, you have to copy the file WM8960.py to the board's file system. The file is located in micropython-lib at https://github.com/micropython/micropython-lib/tree/master/micropython/drivers/codec/wm8960. You can use any suitable tool, e.g. mpremote, to upload the file to the board.

**Comment by teddokano** [comment]
Thank you for answer and showing me location of the file. 
I could upload the file and make the sample run.
